tpp/src/main/TokenClient.js

```diff
@@ -195,6 +198,31 @@ export class TokenClient extends Core {
         });
     }
 
+    /**
+     * Creates a createBulkTransferTokenRequestBuilder for a bulk transfer token request.
+     *
+     * @param transfers List of transfers
+     * @param totalAmount Total amount irrespective of currency. Used for redundancy check.
+     * @param source Source account for all transfer
+     * @returns Builder instance
+     */
+    createBulkTransferTokenRequest(
+        transfers: Array<BulkTransferBodyTransfers>,
+        totalAmount: string | number,
+        source: TransferEndpoint
```

Comment:
Remove this, as discussed.

Reply:
in JS, we can leave it there because we don't override for the case where this exists.